### ISSUE TITLE
Add support for /playlist

### ISSUE THREAD

**Issue by samuel-p** [pluja/Parasitter#164]
**Is your feature request related to a problem? Please describe.**
Yotter does not support the `/playlist` path of YouTube yet. It would be awesome if Yotter would support it.

**Describe the solution you'd like**
Show all videos of a playlist, like YouTube does. For example https://www.youtube.com/playlist?list=PLDIoUOhQQPlXzhp-83rECoLaV6BwFtNC4

**Comment by pluja** [comment]
Yes! This is a planned feature and should not be very difficult to implement. Right now I don't have any time to code, but I hope that next month I will be able to start coding and getting new features to Yotter :D

**Comment by FireMasterK** [comment]
I got this request from a friend aswell 👍
